helm_deploy/values-dev.yaml

```diff
@@ -12,6 +12,10 @@ image:
 ingress:
   enabled: true
   enable_whitelist: true
+  annotations:
+    kubernetes.io/ingress.class: "nginx"
+    certmanager.k8s.io/cluster-issuer: letsencrypt-production
+    nginx.ingress.kubernetes.io/limit-rpm: "60"
```

Comment:
We only have one client, how many requests per minute do we expect from Oasys and how does it cope with back pressure?

Reply:
Yeah I lifted from offender-assessments-api-kotlin. I guess we don't need the rate line? I'll remove for now 👍